services/systemd/lighthouse.service

```diff
@@ -1,14 +1,13 @@
 [Unit]
 Description=Lighthouse Consensus Client
-After=network-setup.service persistence-setup.service
-Wants=network-setup.service persistence-setup.service
-ConditionPathExists=/persistent
+After=network.target network-setup.service persistent-mount.service
```

Comment:
Will we need separate configs for bob and bnet lighthouse?

Reply:
I can't think of anything that would require us to rewrite this file for a separate image, especially since systemd lets you extend existing services to refine their boot order.